indigo/distribution.yaml

```diff
@@ -8675,6 +8675,14 @@ repositories:
       url: https://github.com/So-Cool/report_card.git
       version: master
     status: maintained
+  repository_name:
```

Comment:
Please use a "better" repository name here. Since it contains only one package probably the name of the package.